### ISSUE TITLE
** wildcards unexpected behavior

### ISSUE THREAD

**Issue by guoyunhe** [crowdin/crowdin-cli-2#36]
```yaml
"api_key_env": CROWDIN_API_KEY
"project_identifier": "kdeorg"

files: [
 {
  "source" : "/templates/**/*.pot",
  "translation" : "/%locale_with_underscore%/**/%file_name%.po",
 }
]
```

For example, the source file is `template/messages/kde-workspace/user_manager.pot` and the translation file is `zh_CN/messages/kde-workspace/user_manager.po`.

But when I upload translation, I got this error:
```
Translation file '/home/sign/git/kde/kde-cn/stable/zh_CN/messages/kde-workspace/user_manager.pot/user_manager.po' does not exist
```

It seems `**` is matching both folders and original file names. But from Crowdin document, it should only match folder.

**Comment by guoyunhe** [comment]
More information:

I can upload translations successfully with the old Ruby cli tool. So I guess this is a valid bug of the new Java cli tool. (version 2.0.14, rpm)

**Comment by Andrulko** [comment]
Hi @guoyunhe,

Reviewing this problem and will get back to you with a solution soon!

**Comment by ihor-popyk** [comment]
#40

**Comment by ihor-popyk** [comment]
Hi @guoyunhe 
So, this problem will be fixed in 2.0.15 version.
Thanks for report!
